src/Sulu/Bundle/PageBundle/Resources/js/containers/Form/fields/SegmentSelect.js

```diff
@@ -0,0 +1,43 @@
+// @flow
+import React from 'react';
+import {computed} from 'mobx';
+import {SingleSelect} from 'sulu-admin-bundle/components';
+import type {FieldTypeProps} from 'sulu-admin-bundle/types';
+import {translate} from 'sulu-admin-bundle/utils';
+import webspaceStore from '../../../stores/webspaceStore';
+
+export default class SegmentSelect extends React.Component<FieldTypeProps<string>> {
+    @computed get webspace() {
+        const {formInspector} = this.props;
+
+        return webspaceStore.getWebspace(formInspector.metadataOptions.webspace);
+    }
+
+    handleChange = (value: string) => {
+        const {onChange, onFinish} = this.props;
+
+        onChange(value);
+        onFinish();
+    };
+
+    render() {
+        const {disabled, value} = this.props;
+
+        return (
+            <SingleSelect
+                disabled={!!disabled}
+                onChange={this.handleChange}
+                value={value}
+            >
+                <SingleSelect.Option>
```

Comment:
this is only displayed if i explicitly select it. after saving. it shows `please choose` again. i think the problem is that the `single_select` uses an empty string instead of `null` as default value

Reply:
Hm, that's weird... You are talking about [this line](https://github.com/sulu/sulu/blob/b48343f/src/Sulu/Bundle/PageBundle/Content/Types/SingleSelect.php#L25), right? We might be able to fix that in this `SegmentSelect` component, but that would feel more like a workaround... I tend to change the default value in the line mentioned before, because that would fix the problem in general. Or do you think that is too much of a BC break? :thinking:

Reply:
Have implemented it for now like that.

Reply:
no, changing the value is okay for me 🙂